modules/c++/re/source/RegexPCRE.cpp

```diff
@@ -20,39 +20,41 @@
  *
  */
 
-#if defined(USE_PCRE)
+#include "re/Regex.h" // this has to come before the #ifdef checks below
```

Comment:
Doesn't matter that much I guess but why is it the case that this include has to come before the `#ifdef` check?